### ISSUE TITLE
Snuffler crashing on resize

### ISSUE THREAD

**Issue by bvicic** [pyrocko/pyrocko#341]
Dear Pyrockos. I was trying to resize the window of Snuffler with multiple/one/zero traces opened which caused Snuffler to crash with Segmentation error 11.

I installed pyrocko with anaconda on macos. The only change i did was modifying $HOME/.pyrocko/config.pf with new path to the GFs.

Thanks!
Blaz

**Comment by bvicic** [comment]
when i try to resize the window dragging from a corner.

**Comment by bvicic** [comment]
macos mojave 10.14

pyqt and pyrocko
5.6
2018.11.06

python
Python 3.6.7 | packaged by conda-forge | (default, Nov 20 2018, 18:20:05)

**Comment by miili** [comment]
Did you install pyrocko through conda package manager?

**Comment by bvicic** [comment]
Yes. As written on the pyrocko webpage. I created a fresh env and did all from scratch.
